src/lib/components/ui/button.svelte

```diff
@@ -36,14 +38,15 @@
 		class: className = '',
 		leftAlign = false,
 		children,
+		mode,
 		...restProps
 	}: Props = $props()
 </script>
 
 <span class={`root ${className}`} class:disabled>
 	<svelte:element
 		this={href ? 'a' : 'button'}
-		class={`${dimension} ${variant}`}
+		class={`${dimension} ${variant} ${mode ? 'mode-' + mode : ''}`}
```

Comment:
```suggestion
		class={`${dimension} ${variant} ${mode ? `mode-${mode}` : ''}`}
```

But I'd prefer to use the mode directly and have it initialized to `'auto'` which would.
```suggestion
		class={`${dimension} ${variant} ${mode}`}
```